packages/features/ee/teams/components/MemberInvitationModal.tsx

```diff
@@ -259,6 +262,33 @@ export default function MemberInvitationModal(props: MemberInvitationModalProps)
                 </Button>
               </div>
             )}
+            {modalImportMode === "ORGANIZATION" && (
+              <Controller
+                name="emailOrUsername"
+                control={newMemberFormMethods.control}
+                rules={{
+                  required: t("enter_email_or_username"),
+                }}
+                render={({ field: { onChange, value } }) => (
+                  <>
+                    <TeamInviteFromOrg
+                      selectedEmails={value}
+                      handleOnChecked={(userEmail) => {
+                        // If 'value' is not an array, create a new array with 'userEmail' to allow future updates to the array.
+                        // If 'value' is an array, update the array by either adding or removing 'userEmail'.
+                        const newValue = Array.isArray(value)
```

Comment:
```suggestion
                        const emailArray = Array.isArray(value) ? value : [value];
                        const newValue = emailArray.includes(userEmail) ? emailArray.filter(email => email !== userEmail) : [...emailArray, userEmail];
```

Better even is to extract it;

```javascript
function toggleElementInArray(value: string[] | string, element: string): string[] {
  const array = Array.isArray(value) ? value : [value];
  return array.includes(element) ? array.filter(item => item !== element) : [...array, element];
}
```